Adding static map sample - Solves #222

hhkaos opened the pull request.

goldenlimit commented on web-js/create-static-map-image/index.html:
```diff
@@ -0,0 +1,295 @@
+        </div>
+
+        <div class="form-group locationType" id="typeAddress" style="display:none">
+            <label class="col-sm-3 control-label">Dirección:</label>
```
Correct this to "Direction"

goldenlimit commented:
@hhkaos 
Looks good. Couple of small things if we can fix it before we merge this request:

1. Udpate url to 3.16 js api
2. When change "Location type" from latitude&longitude to address, seems the current logic must clear the latitude and longitude value, then the address location can read, otherwise the code will automatically assume the input as lat&long. As a demo code, you don't have to change this logic, but its better to mentioned in the README file 

Good job! :+1:

hhkaos commented:
Sorry @goldenlimit for the delay. I fixed everything you said :). And thanks, I fixed the generator also, the reason it wasn't working was a bug, thx! :dancer:
